jest.setup.js

```diff
@@ -1,21 +1,38 @@
-import { setUpTests } from 'react-native-reanimated/lib/reanimated2/jestUtils';
+// import '@testing-library/jest-native/extend-expect' and export it
+// to make it available in all tests
+import '@testing-library/jest-native/extend-expect';
 
-// configure jest for react native reanimated
-// setUpTests();
+// mock async storage
+jest.mock('@react-native-async-storage/async-storage', () =>
+  require('@react-native-async-storage/async-storage/jest/async-storage-mock')
+);
 
-// mock registerRootComponent to resolve error
-// jest.mock('expo');
+export * from '@testing-library/react-native';
 
-// mock useFonts so that it is loaded right away
+// mock native animated helper to resolve useNativeDriver error
+jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');
+
+// // mock registerRootComponent to resolve error
+// jest.mock('expo', () => ({
+//   registerRootComponent: jest.fn(),
+// }));
+
+// // mock useFonts so that it is loaded right away
 // jest.mock('@expo-google-fonts/comfortaa', () => ({
 //   // useFonts will return [true] right away
 //   useFonts: ({}) => [true],
 // }));
```

Comment:
commented out?

Reply:
yeah leaving the code there so that in the future when we need to mock those files we can uncomment it